CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Adds
+
+* Adds a `preventUpdateAssets` to the module that will not try to update already existing assets on import.
```

Comment:
You are right to want this to change, because it is doing a great deal of extra work, but the actual fix is different... attachments are immutable, except for crops, docIds and archivedDocIds. Therefore the file itself should never need to be re-uploaded, although some crops present in the imported database might need to be inserted, which I don't see happening in the existing code.

So I will discuss with the team, I think this would make more sense as a warranty issue in gitlab.

My fault perhaps for making attachments do so much complicated bookkeeping to avoid copies, but copies are very expensive when multiplied across all modes and locales.